src/main/java/edu/greenblitz/robotName/commands/CollectNote.java

```diff
@@ -11,10 +12,13 @@ public class CollectNote extends ConditionalCommand {
 
     public CollectNote() {
         super(
-                new NoteFromIntakeToShooter(),
+                new NoteToIntake()
+                        .andThen(new MoveArmToIntakePosition())
+                        .alongWith(new NoteFromIntakeToShooter()),
                 new NoteToIntake()
```

Comment:
you call this twice?

Reply:
it is a conditional command, first call is if shooter is at angle, second if not